documentation/src/main/resources/pages/ditto/installation-operating.md

```diff
@@ -49,13 +61,19 @@ ditto.gateway.authentication {
 }
 ```
 
+In order to do this by specifying a Java system property, use the following:
+
+```
+-Dditto.gateway.authentication.oauth.openid-connect-issuers.myprovider=https://localhost:9000/
```

Comment:
With #535 we need to omit https:// in the issuer configuration

Reply:
indeed